pom.xml

```diff
@@ -62,7 +62,8 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins-core.version>2.58-20170501.155158-2</jenkins-core.version> <!-- TODO: Switch to release once https://github.com/jenkinsci/jenkins/pull/2730 is merged and released -->
+        <jenkins-war.version>2.58-20170501.155220-2</jenkins-war.version> <!-- TODO: Switch to release once https://github.com/jenkinsci/jenkins/pull/2730 is merged and released -->
```

Comment:
Yup, is good, I just screwed up and ran it with Java 7 initially, which silently didn't run tests without saying why. Good times, good times.